src/server/enrollment-system.js

```diff
@@ -1039,7 +1039,12 @@ function veteranToDemographicsInfo(veteran) {
           addressTypeCode: 'P',  // TODO(awong): this code is from VHA Standard Data Service (ADRDEV01) Address Type List P==Permanent. Determine if we need it.
         },
         emails: {
-          email: veteran.email,
+          email: [
+            {
+              address: veteran.email,
+              type: '1'
```

Comment:
I think those are the right things to reference. You can also access the VHA SDS directly, but I'm not sure if you'll be able to access it outside of the VA network: http://vaausesrapp80.aac.va.gov:7404/ds/List/EmailAddressType